spec/models/conversion_host/configurations_spec.rb

```diff
@@ -75,7 +75,7 @@
         task_id = described_class.enable_queue(params)
         expect(MiqTask.find(task_id)).to have_attributes(:name => expected_task_action)
         expect(MiqQueue.first).to have_attributes(
-          :args        => [params],
+          :args        => [params.merge(:task_id => task_id)],
```

Comment:
can you double check this? I don't see why `params` will be tampered with in the course of action

Reply:
Good question. If you remove `params.symbolize_keys` in the `enable_queue` method, it doesn't happen. As to why that would alter the behavior, I'm not sure right off.